packages/wallet/src/redux/__tests__/test-scenarios.ts

```diff
@@ -86,6 +91,10 @@ export const gameCommitment1: Commitment = {
   allocation: ['0x05', '0x05'],
   destination: [],
 };
+export const signedCommitment19 = {
```

Comment:
I was actually thinking that this could be the new standard. It's useful to have the `turnNum` in the name and you can usually deduce the state type from it. (We might need to have `signedCommitment19c` for conclude states.) I will make further changes in a future PR.